api-examples/drain_conn/main.go

```diff
@@ -31,15 +31,18 @@ func main() {
 		log.Fatal(err)
 	}
 
+	// Just to not collide using the demo server with other users.
+	subject := nats.NewInbox()
```

Comment:
Here and everywhere where you changed that, I think that it may break the examples that could be run together. For instance one of the example subscribe, one publishes and with this change the sub won't get the message.
Also, I believe that these were originally used for the doc, but I may be wrong (cc'ed @sasbury to confirm)

Reply:
ok, understand what you mean for the pair examples. Will revisit.

Reply:
This one self contained, so ok. But again I will look for the "pair" examples and revert.